Sources/Sentry/SentrySessionReplay.m

```diff
@@ -289,13 +292,18 @@ - (void)captureSegment:(NSInteger)segment
     replayEvent.segmentId = segment;
     replayEvent.timestamp = videoInfo.end;
 
-    __block NSArray<SentryRRWebEvent *> *events;
+    __block NSArray<SentryBreadcrumb *> *breadcrumbs;
+    [SentrySDK.currentHub
+        configureScope:^(SentryScope *_Nonnull scope) { breadcrumbs = scope.breadcrumbs.copy; }];
```

Comment:
`m`: You don't need to copy the array as `scope.breadcrumbs` returns a new array anyways https://github.com/getsentry/sentry-cocoa/blob/3478fc541487bfd77886aa7d430d6242b9a6c679/Sources/Sentry/SentryScope.m#L204-L220

Reply:
```suggestion
        configureScope:^(SentryScope *_Nonnull scope) { breadcrumbs = scope.breadcrumbs; }];
```